app/backend/aquifers/filters.py

```diff
@@ -0,0 +1,41 @@
+"""
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+from django.contrib.gis.geos import GEOSException, Polygon
+from django.contrib.gis.gdal import SpatialReference, CoordTransform
+from rest_framework.filters import BaseFilterBackend
+
+
+class BoundingBoxFilterBackend(BaseFilterBackend):
+    """
+    Filter that allows geographic filtering with a bounding box.
+    """
+
+    def filter_queryset(self, request, queryset, _view):
+        sw_long = request.query_params.get('sw_long')
+        sw_lat = request.query_params.get('sw_lat')
+        ne_long = request.query_params.get('ne_long')
+        ne_lat = request.query_params.get('ne_lat')
+
+        if sw_long and sw_lat and ne_long and ne_lat:
+            ct = CoordTransform(SpatialReference('NAD83'), SpatialReference('EPSG:4326'))
```

Comment:
@stephenhillier is this the right kind of transform to use here? It seems to work but I am assuming that the query string parameters are in 4326 and it looks like the geom data in the DB for aquifers is NAD83. It works for the local aquifer fixtures but I don't know for sure if this is doing the right thing. Some transform is needed because without it every bounding box query string I tried returns no aquifers.

Reply:
The aquifer layer geometry uses SRID 3005 (BC Albers, which uses metres as a base unit) while the coordinates in the query (`sw_long`, `sw_lat` etc) are SRID 4326 (and are in degrees). By specifying that the bounding box is SRID 4326, GeoDjango can transform the bounding box to 3005 to do the intersect.

With that in mind, just setting the `srid` property on the polygon you made from the bounding box (i.e. `bbox.srid = 4326`, without the `bbox.transform(ct)` statement) seems more like what you need to do here. However... there are people more knowledgeable than me who can clarify this for us next week.

Reply:
Yes that change worked well - thx!